src/lib/components/globalhealthbar/GlobalHealthBar.component.js

```diff
@@ -0,0 +1,139 @@
+//@flow
+import React from 'react';
+import VegaChart from '../vegachart/VegaChart.component.js';
+
+type Props = {
+  id: string,
+  alertSegments: {
+    severity: 'critical' | 'warning' | 'none',
+    startSegment: string,
```

Comment:
I'm not sure we really want to pass a startSegment and endSegment, maybe giving the alert object directly makes more sense ? It would avoid each component using this component to reformat the alerts before passing them to the global health component

Reply:
The global health component would then be responsible of segment splitting